src/styles/card.scss

```diff
@@ -0,0 +1,108 @@
+@import "../styles/utils/mixins.scss";
+
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  max-width: 200px;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+
+  font-weight: 400;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    padding: 0;
+    margin-bottom: 40px;
+  }
+
+  &__product-image {
+    object-fit: cover;
+    width: 100%;
+    height: 100%;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $color-main-accent;
+
+    &:hover {
+      color: $color-card-title-hover;
+    }
+  }
+
+  &__code {
+    padding: 4px 0 0;
+
+    font-size: 10px;
+    color: $color-secondary;
+    line-height: 14px;
+  }
+
+  &__button {
+    all: unset;
+
+    box-sizing: border-box;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 166px;
+    height: 40px;
+    border: 1px solid transparent;
+    border-radius: 5px;
+
+    font-size: 14px;
+    color: #fff;
+    font-weight: 700;
+    text-transform: uppercase;
+    background-color: $color-blue-accent;
+    cursor: pointer;
+
+    &:hover {
+      border-color: $color-blue-accent;
+
+      background-color: #fff;
+      color: $color-blue-accent;
+    }
+  }
+
+  .rate {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding: 16px 0;
+  }
+
+  .reviews {
+    font-size: 10px;
+    color: $color-main-accent;
+  }
+
+  .price {
+    display: flex;
```

Comment:
If they are separate blocks, move them to separate files
But I'd better made them as elements of `card` block